coderd/database/queries/auditlogs.sql

```diff
@@ -1,127 +1,114 @@
-    users.deleted AS user_deleted,
-    users.quiet_hours_schedule AS user_quiet_hours_schedule,
-    COALESCE(organizations.name, '') AS organization_name,
-    COALESCE(organizations.display_name, '') AS organization_display_name,
-    COALESCE(organizations.icon, '') AS organization_icon,
-    COUNT(audit_logs.*) OVER () AS count
-FROM
-    audit_logs
-    LEFT JOIN users ON audit_logs.user_id = users.id
-    LEFT JOIN
-        -- First join on workspaces to get the initial workspace create
-        -- to workspace build 1 id. This is because the first create is
-        -- is a different audit log than subsequent starts.
-        workspaces ON
-		    audit_logs.resource_type = 'workspace' AND
-			audit_logs.resource_id = workspaces.id
-    LEFT JOIN
-	    workspace_builds ON
-            -- Get the reason from the build if the resource type
-            -- is a workspace_build
-            (
-			    audit_logs.resource_type = 'workspace_build'
-                AND audit_logs.resource_id = workspace_builds.id
-			)
-            OR
-            -- Get the reason from the build #1 if this is the first
-            -- workspace create.
-            (
-				audit_logs.resource_type = 'workspace' AND
-				audit_logs.action = 'create' AND
-				workspaces.id = workspace_builds.workspace_id AND
-				workspace_builds.build_number = 1
-			)
-		LEFT JOIN organizations ON audit_logs.organization_id = organizations.id
+SELECT sqlc.embed(audit_logs),
+	   -- sqlc.embed(users) would be nice but it does not seem to play well with
+	   -- left joins.
+	   users.username                           AS user_username,
+	   users.name                               AS user_name,
+	   users.email                              AS user_email,
+	   users.created_at                         AS user_created_at,
+	   users.updated_at                         AS user_updated_at,
+	   users.last_seen_at                       AS user_last_seen_at,
+	   users.status                             AS user_status,
+	   users.login_type                         AS user_login_type,
+	   users.rbac_roles                         AS user_roles,
+	   users.avatar_url                         AS user_avatar_url,
+	   users.deleted                            AS user_deleted,
+	   users.quiet_hours_schedule               AS user_quiet_hours_schedule,
+	   COALESCE(organizations.name, '')         AS organization_name,
+	   COALESCE(organizations.display_name, '') AS organization_display_name,
+	   COALESCE(organizations.icon, '')         AS organization_icon
+FROM audit_logs
+		 LEFT JOIN users ON audit_logs.user_id = users.id
+		 LEFT JOIN organizations ON audit_logs.organization_id = organizations.id
+	-- First join on workspaces to get the initial workspace create
+	-- to workspace build 1 id. This is because the first create is
+	-- is a different audit log than subsequent starts.
+		 LEFT JOIN workspaces ON
+	audit_logs.resource_type = 'workspace' AND
+	audit_logs.resource_id = workspaces.id
```

Comment:
The indent formatting in this whole file seems messed up now

Reply:
Fixed